extensions/kube-object-event-status/renderer.tsx

```diff
@@ -0,0 +1,42 @@
+import { LensRendererExtension, K8sApi } from "@k8slens/extensions";
+import { resolveStatus, resolveStatusForCronJobs, resolveStatusForPods } from "./src/resolver"
+
+export default class EventResourceStatusRendererExtension extends LensRendererExtension {
+  kubeObjectStatusTexts = [
+    {
+      kind: "Pod",
+      apiVersions: ["v1"],
+      resolve: (object: K8sApi.Pod) => resolveStatusForPods(object)
+    },
+    {
+      kind: "ReplicaSet",
+      apiVersions: ["v1"],
+      resolve: (object: K8sApi.KubeObject) => resolveStatus(object)
```

Comment:
Why some of these use specific types and other use generic `KubeObject`?

Reply:
`resolveStatusFromPods` and `resolveStatusFromCronJobs` requires specific type. I've changed this now to use always specific type to be more consistent.